src/rules/store/on-function-explicit-return-type.ts

```diff
@@ -54,10 +54,16 @@ export default createRule<Options, MessageIds>({
 })
 
 function getFixes(
-  { params }: TSESTree.ArrowFunctionExpression,
+  node: TSESTree.ArrowFunctionExpression,
   sourceCode: Readonly<TSESLint.SourceCode>,
   fixer: TSESLint.RuleFixer,
 ) {
+  const { params } = node;
+    
+  if (params.length === 0) {
```

Comment:
Thanks for creating the PR!
The implementation looks good, but it would be awesome if we could also include a test case.
This can be the code snippet that's posted in the issue.

Reply:
Added test and found/fixed an issue